### PR TITLE
Default values for 2 columns widget

### LINKED ISSUE #1517: Tweaks to conditions tab styling
Our problems: 

- https://github.com/WeakAuras/WeakAuras2/issues/1516

As a first measure, rename all Glow Conditions to have the same name as their `Display` tab counterpart

- `Visibility` => `Show Glow`

### PR DESCRIPTION
**Pull request by mrbuds**
# Description
Allow set a default value for the 2 columns widget by setting "true" as 3rd element of "display" table

Fixes #1517

### CODE REVIEW

**Comment by Stanzilla** [comment]
hello darkness my old friend, says the minor version number

File: WeakAuras/RegionTypes/Icon.lua [hunk 1]
```diff
@@ -69,7 +69,7 @@ local properties = {
     default = 32
   },
   glow = {
-    display = { WeakAuras.newFeatureString .. L["Glow"], L["Show Glow"] },
+    display = { WeakAuras.newFeatureString .. L["Glow"], L["Show Glow"], true },
```
Comment (emptyrivers) [inline WeakAuras/RegionTypes/Icon.lua]: I dislike this for religious reasons, since display names don’t have anything to do with defaults. Is there any way that we can set the default flag on something other than the display name?

Comment (emptyrivers) [reply WeakAuras/RegionTypes/Icon.lua]: Also some day in the distant future we may have a use for a dropdown tree that is three or more levels deep, and this interferes with that.

Comment (mrbuds) [reply WeakAuras/RegionTypes/Icon.lua]: i didn't find how to pass along more properties to the ace widget

Comment (InfusOnWoW) [reply WeakAuras/RegionTypes/Icon.lua]: That isn't ace though, that's the property definitions which get converted to ace tables in ConditionOptions.lua. 

The code is pretty hard to follow though, and it's called display there too....
